### DISCUSSION TITLE
How to disable software updates

### DISCUSSION THREAD

**Discussion by redxtech** [cockpit-project/cockpit#19895] [Q&A] [answered]
I'm running cockpit on NixOS, where PackageKit isn't installed, and doesn't work if it is installed, due to NixOS handling packages in a fundamentally different manner.

This results in the software updates module not being functional, and erroring out.

Is there a way for me to disable the software updates section?

**Comment by martinpitt** [comment, marked as answer]
The easiest way, also for an official NixOS package, is to remove /usr/share/cockpit/packagekit/. In other distributions (Fedora, Debian, etc.) that is a separate binary package.

**Reply by redxtech** [reply]
I've made an overlay to remove that folder during the install process, and it works great.
Thank you!

**Comment by maricopajoe** [comment]
How can we get cockpit to work on Nixos?  All the binaries are missing from their package

**Reply by jelly** [reply]
Can you provide more context? The software updates page :100: likely won't work with NixOS as PackageKit does not support that.
